Ethinylestradiol/Ethinylestradiol-Report.md

```diff
@@ -0,0 +1,419 @@
+# Building and evaluation of a PBPK model for Ethinylestradiol in adults
+
+
+
+| Version                                         | 1.1-OSP10.0                                                  |
+| ----------------------------------------------- | ------------------------------------------------------------ |
+| based on *Model Snapshot* and *Evaluation Plan* | https://github.com/Open-Systems-Pharmacology/Ethinylestradiol-Model/releases/tag/v1.0 |
```

Comment:
Hmm, the snapshot did not change, as no changes to the model were required. So the snapshot is the same, but the version of the report is increased because the first version was created with v9. Does it really make sense to create new tags for the snapshots, even if they did not change?

Reply:
No, it does not make sense.
But here you have report version **1.1**-OSP10.0 which refers to the tag **v1.0** which is a mismatch.
So if the report was created from the tag 1.0 (without any changes in snapshot and/or evaluation plan), than the report version should be **1.0**-OSP10.0

Reply:
Ok, I will create new releases with all the typo fixes in the evaluation plans.